exec/java-exec/src/main/java/org/apache/drill/exec/work/metadata/ServerMetaProvider.java

```diff
@@ -76,7 +76,7 @@
       .setReadOnly(false)
       .setGroupBySupport(GroupBySupport.GB_UNRELATED)
       .setLikeEscapeClauseSupported(true)
-      .setNullCollation(NullCollation.NC_AT_END)
+      .setNullCollation(NullCollation.NC_HIGH)
```

Comment:
@jinfengni Agree with Paul. The difference between NC_HIGH and NC_AT_END for the DESC case. I checked that Drill uses NC_HIGH for sorting. Please, see it at [jira](https://issues.apache.org/jira/browse/DRILL-5326?focusedCommentId=15900411&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-15900411), since github doesn't show formatted code correctly in the comments.